Sources/MapboxMaps/Ornaments/ScaleBar/MapboxScaleBarOrnamentView.swift

```diff
@@ -346,7 +346,7 @@ internal class MapboxScaleBarOrnamentView: UIView {
     // MARK: - Convenience Methods
 
     private func usesRightToLeftLayout() -> Bool {
-        return UIView.userInterfaceLayoutDirection(for: semanticContentAttribute) == .rightToLeft
```

Comment:
why did we change this? imo the former is better since we are exposing scale bar view under `OrnamentsManager`, so the client code can always enforce `semanticContentAttributed` on this view.
```
mapView.ornaments.scaleBarView.semanticContentAttribute = .forceLeftToRight
```
when this happens, the former will return false, but the new one will return true when the trait environment is in RTL locale. WDYT?

Reply:
It was my suggestion, using `traitCollection` will allow the scale bar to respect trait collection being overridden upstream. After looking more closely, I found out that neither of two is the recommended way to check layout direction.
`UIView.effectiveUserInterfaceLayoutDirection` is. Probably should switch to using this instead.